cli/version.txt

```diff
@@ -1 +1 @@
-0.1.0-beta.5
+0.2.0-beta.1
```

Comment:
I would keep this at `0.1.0-beta.6` until we are certain we are going to rev the minor version number (e.g. after the terraform or azdo changes land)